README.md

```diff
@@ -37,7 +43,24 @@ Requires a configuration file `configuration.json` with information for each eng
 ```
 If an `executablePath` or `parameterExpression` is omitted, the engine is expected to be hosted externally. An example of this is the `External` engine in the above configuration. Engines can optionally be marked `verbose` to print failed queries while the tests are run from [Run Tests for Engine](#run-tests-for-engine)
 
-##Run Tests for Engine
+`testTimeout`, `testCount`, `testSorting`, `queryComplexity` are all optional configurations.
+`testTimeout` sets the time limit in seconds for the duration of a test (default=30).
+`testCount` limits the number of tests to execute (default=all).
+`testSorting` determines how to sort the tests if `testCount` is set. There are four different sortings:
+* `Random` (default) - Takes `testCount` generated tests randomly
+* `FILO` - Takes the last `testCount` generated tests
+* `FIFO` - Takes the first `testCount` generated tests
+* `Fair` - Takes an equal amount of each test-sort, summing up to no more than `testCount` tests
+
+`queryComplexity` determines the complexity of the queries in the tests (the number of operators).
+Both the upper and lower bound can be set.
+If only one element is in the array the upper bound will be set to that.
+If more than two elements are present those beyond index `1` will be omitted.
+If the array is empty, no bound is set.
+`omitTests` determines if the tests are to be executed. If true, the tests will instead be written to the file in the filepath specified in `testsSavePath`, or the default-path `./${engine.name}_tests` (default=`false`).
```

Comment:
My initial though was that "omitTests" ment that tests would be written to disk. Maybe we should call it "omitTesting"?

Reply:
It would also be nice if we could write the tests to disk and still run them

Reply:
So, after giving it some thought I've come up with a few ideas:

We can either keep it the way it is implemented right now, making `omitTesting` determine whether we must save the queries, either in a default path, or through `testsSavePath`

Or, we could remove `omitTesting` and filter the testing by both `enabled` and `testsSavePath`. Then we do two checks: One determining if the tests should be executed (`enabled`), and one for saving the queries (`testsSavePath != null`)

We can also add an attribute determining if the queries should be saved (`saveQueries`), but that would maybe make `enabled` somewhat redundant since `enabled = omitTesting || saveQueries`

Personally I am a bigger fan of the second option because I like the idea of fewer attributes, and the implicity of setting `testsSavePath`. What do you think?

Reply:
I agree. The second option sounds a bit simpler for the user. :+1: